app/overrides/spree/admin/locales/show/add_supported_locales.html.erb.deface

```diff
@@ -1,10 +1,10 @@
 <!-- insert_bottom '[data-hook="localization_settings_body"]' -->
 <div class="form-group">
   <label for="supported_locales_">
-    <%= Spree.t(:'i18n.supported_locales') %>
+    <%= I18n.t(:'spree.i18n.supported_locales') %>
```

Comment:
What about using `ActionView::Helpers::TranslationHelper#t` by dropping the `I18n` namespace in the ERB view code?